controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -64,6 +88,22 @@ void wfa_ca::reply_invalid_error(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
     reply(sd, cmdu_tx, reply_string);
 }
 
+bool wfa_ca::reply_running(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx)
+{
+    std::string reply_string = "status," + wfa_ca_status_to_string(eWfaCaStatus::RUNNING);
+    return reply(sd, cmdu_tx, reply_string);
+}
+
+void wfa_ca::reply_complete(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
+                            const std::string &description)
+{
```

Comment:
Since the wfa_ca replies are always of the form `status,STATUS[,reply]`, I think it would be more useful to add the body of this function to reply() itself, and give reply() the signature

```
bool reply(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx, eWfaCaStatus status, const std::string &description = "");
```

With that, the reply_invalid() function is probably no longer useful, you can just call reply() directly. Maybe even reply_running and reply_complete are not so useful (the just save setting the eWfaCaStatus).

(nitpick)

Reply:
changed